Godeps/Godeps.json

```diff
@@ -2997,6 +3013,21 @@
 			"Rev": "20b71e5b60d756d3d2f80def009790325acc2b23"
 		},
 		{
+			"ImportPath": "gopkg.in/square/go-jose.v2",
```

Comment:
go-jose is already vendored at github.com/square/go-jose. Can you just update that?

Reply:
We import `gopkg.in/square/go-jose.v2` in `go-oidc` which I think is the recommended import. https://github.com/square/go-jose#versions

Maybe we can switch the import in k8s.io/kubernetes ?

Reply:
Ya, my preference would be to update existing code to use gopkg.in... and avoid vendoring two of the same library.

Reply:
Gotcha! I can check the places using go-jose.